packages/azure-themes/src/azure/IExtendedSemanticColors.ts

```diff
@@ -1,6 +1,15 @@
 import { ISemanticColors } from 'office-ui-fabric-react';
 
 export interface IExtendedSemanticColors extends ISemanticColors {
+  bodyTextHovered: string;
```

Comment:
Should this be bodyTextHover? I see that other variables use just 'hover' but this one is using 'hovered'. Either way, it's better to have consistent naming.